website/docs/targeting/setting-evaluation.md

```diff
@@ -0,0 +1,157 @@
+:::
+
+### Percentage Evaluation Attribute
+
+The default attribute used for percentage evaluation is the [User Object]'s `Identifier`. However, you can change this to any attribute you want.
+
+*TODO - Where and how to change this*
+
+### Example Scenarios for Percentage-Based Targeting
+
+Imagine you have two users, Jane and Joe, and you're experimenting with two different feature flags (`isTwitterSharingEnabled` and `isFacebookSharingEnabled`) that use percentage-based targeting. In these scenarios, we see how percentage-based targeting allows for a controlled and gradual rollout of features, ensuring a smooth transition for users like Jane and Joe.
+
+First, the users are assigned a number between 0-99 based on the hash of their identifier and the feature flag's key. This number determines their eligibility for a feature flag based on the percentage options set on the Dashboard.
+
+|      | isTwitterSharingEnabled                                                   | isFacebookSharingEnabled                                                    |
+| ---- | ------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| Jane | `hash('Jane' + 'isTwitterSharingEnabled') mod 100` <br/> Results in **8** | `hash('Jane' + 'isFacebookSharingEnabled') mod 100` <br/> Results in **64** |
+| Joe  | `hash('Joe' + 'isTwitterSharingEnabled') mod 100` <br/> Results in **32** | `hash('Joe' + 'isFacebookSharingEnabled') mod 100` <br/> Results in **12**  |
+
+1. **Initial Setting: 0% ON / 100% OFF**
+
+|      | isTwitterSharingEnabled <br/> 0% ON / 100% OFF | isFacebookSharingEnabled <br/> 0% ON / 100% OFF |
+| ---- | ---------------------------------------------- | ----------------------------------------------- |
+| Jane | 8 >= 0 <br/>-> **OFF**                         | 64 >= 0 <br/>-> **OFF**                         |
+| Joe  | 32 >= 0 <br/>-> **OFF**                        | 12 >= 0 <br/>-> **OFF**                         |
+
+2. **Adjustment to 10% ON / 90% OFF**
+In this case both feature flags are enabled for only 10% of users.
+
+|      | isTwitterSharingEnabled <br/> 10% ON / 90% OFF | isFacebookSharingEnabled <br/> 10% ON / 90% OFF |
+| ---- | ---------------------------------------------- | ----------------------------------------------- |
+| Jane | 8 < 10 <br/>-> **ON**                          | 64 >= 10 <br/>-> **OFF**                        |
+| Joe  | 32 >= 10 <br/>-> **OFF**                       | 12 >= 10 <br/>-> **OFF**                        |
+
+:::info
+Despite both feature flags being set to 10% ON / 90% OFF, Jane is only enabled for the `isTwitterSharingEnabled` feature flag. Because Jane and Joe have different identifiers, they're assigned different numbers, which determines their eligibility for each feature flag.
```

Comment:
```suggestion
Despite both feature flags being set to 10% ON / 90% OFF, the `isTwitterSharingEnabled` feature flag is only enabled for Jane. Because Jane and Joe have different identifiers, they're assigned different numbers, which determines their eligibility for each feature.
```